pkg/server/store/pkg/schema.go

```diff
@@ -128,3 +143,40 @@ func updateInstallScript(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusOK)
 	_, _ = w.Write([]byte("{}"))
 }
+
+func cancelHandler(w http.ResponseWriter, r *http.Request) {
+	apiContext := types.GetAPIContext(r.Context())
+	defer r.Body.Close()
+	name := apiContext.Name
+	if err := airgap.CancelDownload(name); err != nil {
+		logrus.Warnf("failed to cancel package %s download, %v", name, err)
+	}
+	w.WriteHeader(http.StatusOK)
+	_, _ = w.Write([]byte("{}"))
```

Comment:
Using `w.Write([]byte(""))` we cannot handle the Action from API-UI. Will got a 500 error when using Send Request
If using `r.WriteResponse(http.StatusOK, types.APIObject{})` wen can get 200 status code and empty body from API-UI.

Reply:
But I am sending `[]byte("{}")` instead of `[]byte{}`, It would be fine.